general-superstaq/general_superstaq/finance.py

```diff
@@ -60,10 +62,11 @@ def read_json_maxsharpe(json_dict: gss.MaxSharpeJson) -> MaxSharpeOutput:
 
 
 class Finance:  # pylint: disable=missing-class-docstring
+    """Overarching Finance class to implement methods above"""
```

Comment:
nit: I think it would be nice to briefly mention the methods being implemented by this overarching Finance class